packages/client/wallets/aa/playwright/tests/functions/aa-wallets.ts

```diff
@@ -0,0 +1,10 @@
+import { Page, expect } from "@playwright/test";
+
+export async function getWallet(page: Page,) {
+    await page.goto("/");
+    await page.getByPlaceholder('Email').fill('begona+testgetwallet@paella.dev');
+    await page.getByPlaceholder('Private key').fill('274c683c7aaa921afb22b49ddaf3395d9f61f72245fd3e1c4885b39caff12ec0');
```

Comment:
I know that the front has an input for introducing a `privateKey`, but that's not used by the developer when implementing AA. So I'm curious about it, why it's used, and its impact on the tests

Reply:
For create an ethers Wallet using the ethers lib you need the pk. The users of course will use the signer itself, but in the demo we need to instantiate the signer.

Reply:
I see, thanks for the explanation. It leads me to this questions:
- I see us instantiating a custom signer, but to test all the paths, we will need to switch between all possible ones ( 
`type Signer = FireblocksNCWSigner | ethers.Signer | Web3AuthSigner;` ) and that isn't happening, right? 
  - Is that because this is a v0?